content/docs/2.6/scalers/datadog.md

```diff
@@ -99,3 +99,11 @@ spec:
     authenticationRef:
       name: keda-trigger-auth-datadog-secret
 ```
+
+## Polling intervals and Datadog rate limiting
+
+[API Datadog endpoints are rate limited](https://docs.datadoghq.com/api/latest/rate-limits/). Depending on the state of the `ScaledObject` there are two different parameters to control how often (per `ScaledObject`) we query Datadog for a metric.
+
+When scaling from 0 to 1, the polling interval is controlled by KEDA, using [the `spec.pollingInterval` parameter in the `ScaledObject` definition](https://keda.sh/docs/2.6/concepts/scaling-deployments/#pollinginterval). For example, if this parameter is set to `60`, KEDA will poll Datadog for a metric value every 60 seconds while the number of replicas is 0.
```

Comment:
Encode the reference to the site-local page as a path (ideally a relative path that doesn't mention the docs version), not an external URL.